src/installer/tests/Assets/TestProjects/HostApiInvokerApp/HostFXR.cs

```diff
@@ -24,27 +24,27 @@ internal enum hostfxr_resolve_sdk2_result_key_t : int
                 global_json_path = 1,
             }
 
-            [StructLayout(LayoutKind.Sequential, CharSet = Utils.OSCharSet)]
+            [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
             internal struct hostfxr_dotnet_environment_sdk_info
             {
-                internal int size;
+                internal UIntPtr size;
                 internal string version;
                 internal string path;
             }
 
-            [StructLayout(LayoutKind.Sequential, CharSet = Utils.OSCharSet)]
+            [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
             internal struct hostfxr_dotnet_environment_framework_info
             {
-                internal int size;
+                internal UIntPtr size;
                 internal string name;
                 internal string version;
                 internal string path;
             }
 
-            [StructLayout(LayoutKind.Sequential, CharSet = Utils.OSCharSet)]
+            [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
             internal struct hostfxr_dotnet_environment_info
```

Comment:
The `sdk_count` and `framework_count` are also `size_t` in native, so they should also become `UIntPtr`.